crashDataAnalysisTools/merge_crash_and_segment.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pandas as pd
+import sqlite3 as dbi
+
+
+def merge_crash_and_segment(crashes, roadSegments):
+    '''
+    Inputs: - a csv file containing individual crash records for

```

Comment:
Maybe the function input should be two string file names instead of two csv files?